english/sonic-screwdriver/sonic-screwdriver.md

```diff
@@ -6,4 +6,6 @@ When you open the door back in Londo you realize that you left your trusty sidek
 
 Do you:
 
-[Go back and find the impossible girl?](../impossible-girl/impossible.md)
\ No newline at end of file
+[Answer the sudden knock at the door?](.../knock/answer.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.